### PR TITLE
vendor: update syndtr/gocapability

### PR DESCRIPTION
**Pull request by aanm**
/kind dependency-change
/kind feature

#### What this PR does / why we need it:\

This commit adds the ability to run cri-o with new capabilities
available in the linux kernel:

CAP_PERFMON
CAP_BPF
CAP_CHECKPOINT_RESTORE

```release-note
Add ability to have `CAP_PERFMON`, `CAP_BPF` and `CAP_CHECKPOINT_RESTORE` capabilities in supported kernels.
```

### CODE REVIEW

**Comment by haircommander** [comment]
/ok-to-test
/approve 

thanks for the PR @aanm , I only have one nit: can you change the PR description to cri-o, not containerd :)

**Review by mrunalp — APPROVED** [review]

**Comment by aanm** [comment]
> /ok-to-test
> /approve
> 
> thanks for the PR @aanm , I only have one nit: can you change the PR description to cri-o, not containerd :)

@haircommander ups, sorry about that :sweat_smile: done!

**Comment by haircommander** [comment]
no problem at all :smile: 

/lgtm

**Comment by aanm** [comment]
@haircommander is the CI failing because of `runc`. I can see that `runc` hasn't made a release with the following change https://github.com/opencontainers/runc/commit/6dfbe9b80707 so the packets installed with `dnf` in the CI might not have this change.

**Comment by haircommander** [comment]
hm we should be building off of main runc branch for everything except the pipeline tests and the `ci/prow` tests. and it seems integration_Cgroup_v2 is a flake (for this run at least)

/retest

**Review by saschagrunert — APPROVED** [review]

**Comment by haircommander** [comment]
/hold

sorry @aanm , I think we need to hold this until runc bumps (our CI uses master, but upstream k8s CI uses a pinned version) ref: https://github.com/cri-o/cri-o/pull/4467

**Comment by aanm** [comment]
@haircommander do you know anyone from runc to make a new release? I can see the last version was last August so it would probably make sense to create a new one I say.

**Comment by haircommander** [comment]
I do! @kolyshkin and @mrunalp are both maintainers. I thought there was an issue around v1.0.0-rc93 but I can't find it

**Comment by haircommander** [comment]
ah, you can track progress here https://github.com/opencontainers/runc/milestone/10

**Comment by aanm** [comment]
Fixed in https://github.com/cri-o/cri-o/pull/5241

**Pull request closed without merging**
